### PR TITLE
controllers: correct logging of verifyNoStorageConsumerExist func

### PR DESCRIPTION
**Pull request by iamniting**
The func verifyNoStorageConsumerExist is called at 2 places and one of
them is uninstall and the other one is not. Using uninstall keyword
create a confusion among the user.

Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=2062990

### CODE REVIEW

File: controllers/storagecluster/uninstall_reconciler.go [hunk 1]
```diff
@@ -272,10 +272,10 @@ func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.
 	}
 
 	if len(storageConsumers.Items) != 0 {
-		err = fmt.Errorf("Failed to uninstall storage cluster. StorageConsumers are present in the %s namespace. "+
-			"Clean all StorageConsumers for the uninstallation to proceed", instance.Namespace)
-		r.recorder.ReportIfNotPresent(instance, corev1.EventTypeWarning, "", err.Error())
-		r.Log.Error(err, "Uninstall: Waiting for StorageConsumers to be deleted via user.")
+		err = fmt.Errorf("Failed to cleanup Provider resource. StorageConsumers are present in the %s namespace. "+
```
Comment (nb-ohad) [inline controllers/storagecluster/uninstall_reconciler.go:275]:
```suggestion
		err = fmt.Errorf("Failed to cleanup provider resources. StorageConsumers are present in the %s namespace. "+
```

File: controllers/storagecluster/uninstall_reconciler.go [hunk 2]
```diff
@@ -272,10 +272,10 @@ func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.
...
+			"Clean all StorageConsumers for the provider cleanup to proceed", instance.Namespace)
```
Comment (nb-ohad) [inline controllers/storagecluster/uninstall_reconciler.go:276]:
```suggestion
			"Offobard all consumer clusters for the provider cleanup to proceed", instance.Namespace)
```

File: controllers/storagecluster/uninstall_reconciler.go [hunk 3]
```diff
@@ -272,10 +272,10 @@ func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.
...
+		r.recorder.ReportIfNotPresent(instance, corev1.EventTypeWarning, "ProviderCleanup", err.Error())
+		r.Log.Error(err, "Waiting for StorageConsumers to be deleted via user.")
```
Comment (nb-ohad) [inline controllers/storagecluster/uninstall_reconciler.go:278]:
```suggestion
		r.Log.Error(err, "Waiting for all consumer clusters to offboard.")
```

File: controllers/storagecluster/uninstall_reconciler.go [hunk 4]
```diff
@@ -272,10 +272,10 @@ func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.
 	}
 
 	if len(storageConsumers.Items) != 0 {
-		err = fmt.Errorf("Failed to uninstall storage cluster. StorageConsumers are present in the %s namespace. "+
-			"Clean all StorageConsumers for the uninstallation to proceed", instance.Namespace)
-		r.recorder.ReportIfNotPresent(instance, corev1.EventTypeWarning, "", err.Error())
-		r.Log.Error(err, "Uninstall: Waiting for StorageConsumers to be deleted via user.")
+		err = fmt.Errorf("Failed to cleanup provider resources. StorageConsumers are present in the %s namespace. "+
+			"Offobard all consumer clusters for the provider cleanup to proceed", instance.Namespace)
```
Comment (nb-ohad) [inline controllers/storagecluster/uninstall_reconciler.go:276]:
```suggestion
			"Offboard all consumer clusters for the provider cleanup to proceed", instance.Namespace)
```

**Comment by iamniting** [comment]
Manual backport https://github.com/red-hat-storage/ocs-operator/pull/1588

**Pull request merged**